doc/changelog.rst

```diff
@@ -15,13 +15,14 @@ Breaking Changes in 4.0
 .......................
 
 - Removed support for Python 2.7, 3.4, and 3.5. Python 3.6+ is now required.
-- Removed :meth:`~pymongo.database.Database.eval`,
-  :data:`~pymongo.database.Database.system_js` and
-  :class:`~pymongo.database.SystemJS`.
+- Removed :meth:`pymongo.database.Database.eval`,
+  :data:`pymongo.database.Database.system_js` and
+  :class:`pymongo.database.SystemJS`.
 - Removed :meth:`pymongo.mongo_client.MongoClient.fsync`,
   :meth:`pymongo.mongo_client.MongoClient.unlock`, and
   :attr:`pymongo.mongo_client.MongoClient.is_locked`.
-- Removed :mod:`~pymongo.thread_util`.
+- Removed :meth:`pymongo.collection.Collection.parallel_scan`.
+- Removed :mod:`pymongo.thread_util`.
```

Comment:
When did this happen?

Reply:
In https://jira.mongodb.org/browse/PYTHON-2462.